### DISCUSSION TITLE
Sonoff 2652P dongle Plus - transmit_power?

### DISCUSSION THREAD

**Discussion by boris1000** [Koenkk/zigbee2mqtt#15911] [Devices]
I am just setting up my first Zigbee Network. Completely new with Zigbee.
One thing is sure -> Absolutely impressed by Zigbee2MQTT.
I am using the Sonoff2652P (not -E) stick with the 20221102 firmware. For testing purposes I am just setting up two devices
one Phillips Hue Motion sensor and one Silvercrest (Lidl) Switch button. So far all OK and working.
Next was playing with the transmit_power option. As I understood the default setting is 5db and the device can handle
up to 20db - Sure I would like to get the most for my money :-)
I just moved the two devices one stock and 25m away, and YES still connected with the default setting.

Silvercrest button reported 10lqi
Hue motion sensor reported 24lqi
-> Next was increasing transmit_power to 20 and check the devices again
Silvercrest button reported 28lqi
Hue motion sensor reported 0lqi it even does not connect on that distance anymore, need to move it half distance
to get it connected again. 
-> Next was decreasing to 12 and later to 9
Hue motion sensor was coming back again an getting better signal and Switch button was getting worse signal.
Just to mention My 2.4 Wifi is on channel 1 till 6 and Zigbee on channel 11, and Yes sure I will add a router later

Questions:
Can it be that some devices getting confused by increased transmit_power, in this case the Hue motion sensor?
Is the standard transmit_power 9 or 5db?
2.4Wifi is on channel 1 till 6, and Zigbee is on channel 11 is that OK?
The USB-dongle antenna should that be vertical or horizontal orientation or doesn´t matter?

**Comment by tsspmq** [comment]
More is not always better, it could be either some other device is ratcheting up to compensate for the "detected interference" or some other kind of distortion from being that strong.  

Many newer wifi routers will do this.   Also keep in mind you do want to only use 20mhz channels in WiFi.

I have four access points, two do not have 2.4 on, two do, those only us channel 6 and are set to the lowest power setting.   Z2M uses channel 11 and my Hue Hub that I am migrating off of use 25.

My subwoofer wireless transmitter that was always on (Polk Audio made one a few years back for them, plugged in back) it just clobbers the spectrum and causes things in that room no matter what channel they use to have issues.   Needless to say, a $19 cable and an hour of fishing the cable up two walls from the basement ended that problem - once I figured out what it was.   And let me tell you that one device caused me more unexplained headaches for a few years before I figured out it was doing it.   Was one of those plug it in, throw behind the AVR and forget about it things.
